src/Microsoft.DotNet.Build.Tasks/PackageFiles/publishtest.targets

```diff
@@ -12,12 +12,6 @@
     <TargetOS Condition="'$(OSGroup)'!='AnyOS'">$(OSGroup)</TargetOS>
   </PropertyGroup>
 
-  <Target Name="RestoreTestRuntimePackage"
```

Comment:
We should consider keeping this, at least for a while for anyone that has a TestRuntimeProjectJson file property defined. That will help other repo's transition to your new model over time.